base/applications/rapps/include/resource.h

```diff
@@ -89,50 +89,47 @@
-#define IDS_INSTALL              102
-#define IDS_UNINSTALL            103
-#define IDS_MODIFY               104
-#define IDS_APPS_COUNT           105
-#define IDS_WELCOME_TITLE        106
-#define IDS_WELCOME_TEXT         107
-#define IDS_WELCOME_URL          108
-#define IDS_INSTALLED            109
-#define IDS_AVAILABLEFORINST     110
-#define IDS_UPDATES              111
-#define IDS_APPLICATIONS         112
-#define IDS_CHOOSE_FOLDER_TEXT   113
-#define IDS_CHOOSE_FOLDER_ERROR  114
-#define IDS_URL_INVALID          115
-#define IDS_APP_REG_REMOVE       116
-#define IDS_INFORMATION          117
-#define IDS_UNABLE_TO_REMOVE     118
-#define IDS_UNABLE_TO_DOWNLOAD   119
-#define IDS_UNABLE_TO_DOWNLOAD2  120
-#define IDS_UNABLE_TO_QUERY_CERT 121
-#define IDS_INTEG_CHECK_TITLE    122
-#define IDS_INTEG_CHECK_FAIL     123
-#define IDS_INTERRUPTED_DOWNLOAD 124
-#define IDS_UNABLE_TO_WRITE      125
-#define IDS_SELECT_ALL           126
-#define IDS_INSTALL_SELECTED     127
-#define IDS_UNABLE_TO_INSTALL    128
-#define IDS_SELECTEDFORINST      129
-#define IDS_MISMATCH_CERT_INFO   130
-#define IDS_UNABLE_PATH          131
-#define IDS_APP_AUTHORS          132
-#define IDS_APP_DISPLAY_DETAILS  133
-#define IDS_APP_DISPLAY_LIST     134
-#define IDS_APP_DISPLAY_TILE     135
-#define IDS_NO_SEARCH_RESULTS    136
+#define IDS_APPS_COUNT           102
+#define IDS_WELCOME_TITLE        103
+#define IDS_WELCOME_TEXT         104
+#define IDS_WELCOME_URL          105
+#define IDS_INSTALLED            106
+#define IDS_AVAILABLEFORINST     107
+#define IDS_UPDATES              108
+#define IDS_APPLICATIONS         109
+#define IDS_CHOOSE_FOLDER_TEXT   110
+#define IDS_CHOOSE_FOLDER_ERROR  111
+#define IDS_URL_INVALID          112
+#define IDS_APP_REG_REMOVE       113
+#define IDS_INFORMATION          114
+#define IDS_UNABLE_TO_REMOVE     115
+#define IDS_UNABLE_TO_DOWNLOAD   116
+#define IDS_UNABLE_TO_DOWNLOAD2  117
+#define IDS_UNABLE_TO_QUERY_CERT 118
+#define IDS_INTEG_CHECK_TITLE    119
+#define IDS_INTEG_CHECK_FAIL     120
+#define IDS_INTERRUPTED_DOWNLOAD 121
+#define IDS_UNABLE_TO_WRITE      122
+#define IDS_INSTALL_SELECTED     123
+#define IDS_UNABLE_TO_INSTALL    124
+#define IDS_SELECTEDFORINST      125
+#define IDS_MISMATCH_CERT_INFO   126
+#define IDS_UNABLE_PATH          127
+#define IDS_APP_AUTHORS          128
+#define IDS_APP_DISPLAY_DETAILS  129
+#define IDS_APP_DISPLAY_LIST     130
+#define IDS_APP_DISPLAY_TILE     131
+#define IDS_NO_SEARCH_RESULTS    132
 
 /* Tooltips */
 #define IDS_TOOLTIP_INSTALL      200
 #define IDS_TOOLTIP_UNINSTALL    201
 #define IDS_TOOLTIP_MODIFY       202
-#define IDS_TOOLTIP_SETTINGS     203
-#define IDS_TOOLTIP_REFRESH      204
-#define IDS_TOOLTIP_EXIT         205
-#define IDS_TOOLTIP_UPDATE_DB    206
+#define IDS_TOOLTIP_SELECT_ALL   203
+#define IDS_TOOLTIP_SETTINGS     204
+#define IDS_TOOLTIP_REFRESH      205
+#define IDS_TOOLTIP_EXIT         206
+#define IDS_TOOLTIP_UPDATE_DB    207
```

Comment:
Do not change all the numbers here! Just add that single line that you want to add at the end! It is not needed to squeeze IDS_TOOLTIP_SELECT_ALL into the middle. Just place it at the end. That will reduce the diff.

Reply:
I placed it corresponding to it's position on the toolbar for readability.

Reply:
I know. But I don't think that is necessary.